src/components/modals/ModalBody.tsx

```diff
@@ -65,11 +70,15 @@ export function ModalBody({ children }: PropsWithChildren<{}>) {
 }
 ModalBody.displayName = 'ModalBody';
 
-const ModalBodyWrapper = styled.div`
+const ModalBodyWrapper = styled.div<Props>`
   position: relative;
   display: flex;
 
-  overflow: hidden;
+  overflow-y: hidden;
+
+  ${({ isLast }) =>
+    isLast &&
+    'border-radius: 0 0 var(--echoes-border-radius-400) var(--echoes-border-radius-400);'}
```

Comment:
these tokens didn't exist, I replaced them with those that do

Reply:
Thanks Copilot I guess